lib/index.js

```diff
@@ -7,7 +7,7 @@ const rules = {
   'no-await-sync-query': require('./rules/no-await-sync-query'),
   'no-debug': require('./rules/no-debug'),
   'no-dom-import': require('./rules/no-dom-import'),
-  'prefer-expect-query-by': require('./rules/prefer-expect-query-by'),
+  'no-get-by-for-absent-elements': require('./rules/no-get-by-for-absent-elements'),
```

Comment:
Naming suggestion: I find "absent" a bit confusing, maybe something like `no-get-by-for-asserting-element-not-present`, which has a more direct relation to the documentation page [Asserting elements are not present](https://testing-library.com/docs/guide-disappearance#asserting-elements-are-not-present). Thoughts?

Reply:
I like it better that way, it's clearer than `no-get-by-for-absent-elements` 🙂

Reply:
Sorry but this name is not correct. As I mentioned before, this rule doesn't always apply to an assertion with `expect` so the rule can't be named like that (I'm adding a deeper review for it as it seems this rule is always checking for assertions). So we would have to go back to its previous name or rename it to something like `no-get-by-for-checking-element-not-present`

Reply:
Yeah sounds good. As long as the rule name gives a good indication of what it does, I'm fine with any name.

I think `no-get-by-for-checking-element-not-present` is also good 👌